ihmc-perception/src/main/java/us/ihmc/perception/realsense/RealsenseDeviceManager.java

```diff
@@ -139,6 +136,10 @@ private String getDeviceInfo(rs2_device rs2Device, int deviceInfoEnum)
 
    private int updateDeviceCount()
    {
+      realsense2.rs2_delete_device_list(devices);
+      devices = realsense2.rs2_query_devices(context, error);
```

Comment:
Feels like this is correct (performing another query when we want to update the device count). Seems to have solved an issue I was having